codec-http/src/main/java/io/netty/handler/codec/spdy/DefaultSpdyHeaders.java

```diff
@@ -27,12 +27,7 @@
 import static io.netty.util.AsciiString.CASE_SENSITIVE_HASHER;
 
 public class DefaultSpdyHeaders extends DefaultHeaders<CharSequence, CharSequence, SpdyHeaders> implements SpdyHeaders {
-    private static final NameValidator<CharSequence> SpdyNameValidator = new NameValidator<CharSequence>() {
-        @Override
-        public void validateName(CharSequence name) {
-            SpdyCodecUtil.validateHeaderName(name);
-        }
-    };
+    private static final NameValidator<CharSequence> SpdyNameValidator = name -> SpdyCodecUtil.validateHeaderName(name);
```

Comment:
Can use method reference here (may be other places too, I didn't look through the whole thing :) )